plans/integration/one-kernel-scenario/main.fmf

```diff
@@ -1,27 +1,41 @@
 discover+:
     filter+:
         - 'tag: one-kernel-scenario'
+
 provision:
     how: libvirt
     develop: true
 
 prepare+:
+    - name: add custom repos
+      how: ansible
+      playbook: tests/ansible_collections/roles/add-custom-repos/main.yml
+      order: 60
     - name: install one kernel
       how: shell
-      script: pytest -svv tests/integration/conversion/custom-repos/one-kernel-scenario/install_one_kernel.py
-      order: 60
+      script: pytest -svv tests/integration/one-kernel-scenario/install_one_kernel.py
+      order: 61
     - name: reboot machine
       how: ansible
       playbook: tests/ansible_collections/reboot.yml
-      order: 61
+      order: 62
     - name: remove other kernels
       how: shell
-      script: pytest -svv tests/integration/conversion/custom-repos/one-kernel-scenario/remove_other_kernels.py
-      order: 62
+      script: pytest -svv tests/integration/one-kernel-scenario/remove_other_kernels.py
```

Comment:
Just a friendly reminder to change those integration paths accordingly with this PR https://github.com/oamg/convert2rhel/pull/343 when it gets merged

Reply:
Thanks! :D